kentico-michaelb/kontent-sample-app-conference-net
# Bug issue #7

### Motivation

Fixes #7 

### Checklist


### How to test

1. Remove a Photo from a Speaker in Kentico Kontent
2. Launch the MVC app and Navigate to the "Speakers" page
3. Click on a Speaker's name to see his/her detail page
